apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -93,10 +95,14 @@ export async function calculatePoolStats({
     ),
   ]);
 
-  const apr =
-    balPriceUSD && tvl && votingShare
-      ? calculateRoundAPR(round, votingShare, tvl, balPriceUSD) * 100
-      : -1;
+  const apr = calculateRoundAPR(round, votingShare, tvl, balPriceUSD);
+
+  if (apr.total === -1 || apr.breakdown.veBAL === -1) {
+    Sentry.captureMessage("vebalAPR resulted in -1", {
+      level: "warning",
+      extra: { balPriceUSD, tvl, votingShare, roundId, poolId, apr },
+    });
+  }
```

Comment:
@ribeirojose sentry warning added here so we can know which pool and on which round the APR wasn't calculated